packages/itwinui-css/backstop/tests/skip-to-content.html

```diff
@@ -70,117 +70,6 @@ <h1>Skip to content link</h1>
               aria-hidden="true"
             ></svg-imodel-hollow>
           </div>
-
-          <div class="iui-page-header-divider"></div>
```

Comment:
A comment about why the header is empty might help.

Reply:
> but why? this page has nothing to do with the header.

This is not a requirement. Just a suggestion, in case someone thinks to readd the other parts of the header.

> also it's not empty, it has the logo which is focusable

By empty I meant it just has the logo.

Reply:
ok, done